src/content/api/payment-requests.mdx

```diff
@@ -0,0 +1,1655 @@
+    The [Line Items](#line-item-model) paid for by the Asset Type.
+  </Property>
+</Properties>
+
+---
+
+## Payment Activity Model
+
+A Payment Activity records a transaction that has happened on a Payment Request.
+Payment Activities are created when a Payment Request has been `created`, `paid`, `refunded`, `cancelled`, or `expired`.
+
+### Attributes
+
+<Properties>
+  <Property name="type" type="string">
+    The [Payment Activity Type](#payment-activity-types).
+  </Property>
+  <Property name="value" type="monetary">
+    The value of the Payment Activity. Must be less than 100000000 and positive.
+  </Property>
+  <Property name="paymentRequestId" type="string">
+    The Payment Request’s id.
+  </Property>
+  <Property name="merchantId" type="string">
+    The Payment Request’s [Merchant](/api/merchants) id.
+  </Property>
+  <Property name="merchantConfigId" type="string">
+    The Payment Request’s [Merchant Config](/api/merchant-configs) id.
+  </Property>
+  <Property name="merchantAccountId" type="string">
+    The Payment Request’s Merchant [Account](/api/accounts) id.
+  </Property>
+  <Property name="merchantName" type="string">
+    The Payment Request’s Merchant name.
+  </Property>
+  <Property name="createdAt" type="timestamp">
+    When the activity was created.
+  </Property>
+  <Property name="createdBy" type="crn">
+    The identity that created the activity.
+  </Property>
+  <Property name="paymentRequestCreatedBy" type="crn">
+    The identity that created the Payment Request.
+  </Property>
+  <Property name="activityNumber" type="bignumber">
+    Unique sequential number for the activity.
+  </Property>
+  <Property name="shortCode" type="string">
+    A shorter id that can be used for up to two years.
+  </Property>
+  <Property name="assetType" type="string">
+    The Asset Type for the `payment` or `refund` activity.
+  </Property>
+  <Property name="external" type="boolean">
+    `true` if the Payment Activity is recording a transaction that occurred outside the Centrapay system.
+  </Property>
+  <Property name="cancellationReason" type="string">
+    The reason that the Payment Request was cancelled. See [Cancellation Reasons](#cancellation-reasons) for possible values.
+  </Property>
+  <Property name="conditionId" type="number">
+    The id of a Condition if the activity was for a Condition being accepted or declined.
+  </Property>
+  <Property name="idempotencyKey" type="string">
+    Required when confirming a Payment Request. This is an identifier from your system to enforce uniqueness.
+  </Property>
+  <Property name="confirmationIdempotencyKey" type="string">
+    Required when refunding a Pre Auth Confirmation. Should be the same as the idempotencyKey used for Confirmation.
+  </Property>
+  <Property name="preAuth" type="boolean">
+    `true` if the related Payment Request is a Pre Auth.
+  </Property>
+</Properties>
+
+---
+
+## Payment Activity Types
```

Comment:
I would have put this and cancellation reasons as H3 as they are part of the Payment Activity model